src/components/Products/Products.tsx

```diff
@@ -0,0 +1,46 @@
+import "./Products.css";
+
+import React, { useEffect, useState } from "react";
+
+interface ProductParams {
```

Comment:
Create separate file (types.ts) to store all the interfaces and types of the specific component.

Reply:
addressed the comment.